### DISCUSSION TITLE
Welcome to space-trading-sim Discussions!

### DISCUSSION THREAD

**Discussion by JammingJammerIsJammingToJam** [JammingJammerIsJammingToJam/space-trading-sim#3] [Announcements]
## 👋 Welcome to the Hell-Hole
  We’re using Discussions as a place to cause long lasting psychological damage
  * Ask questions you’re wondering about (We won't answer)
  * Share ideas (We won't care)
  * Engage with other community members (Why would we talk to you?)
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪 (JK, this is hell not a friend group)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community (We won't read it)

**Comment by AdrianAgache** [comment]
Can u pls add more issues

**Comment by AdrianAgache** [comment]
Guys this has too much potential to waste
